data/storage.json

```diff
@@ -0,0 +1,13 @@
+{
+  "title": "Storage",
+  "ls": "https://storage.spec.whatwg.org/",
+  "wgs": [
+    {
+      "label": "WHATWG",
+      "url": "https://whatwg.org/"
+    }
+  ],
+  "impl": {
+    "chromestatus": 5630353511284736
```

Comment:
Besides [5630353511284736](https://www.chromestatus.com/features/5630353511284736), there are also [4931497563783168](https://www.chromestatus.com/features/4931497563783168) and [5715811364765696](https://www.chromestatus.com/features/5715811364765696).

Reply:
I know but we don't have any way to link to more than one ID for the time being. I created issue #59 to track this. In the meantime, I chose the ID that is closest to the feature of interest here (quota estimates).

Reply:
OK. I'll merge this PR for now.